docker-compose.yml

```diff
@@ -115,10 +115,10 @@ services:
       test: echo stats | nc 127.0.0.1 11211
   redis:
     <<: *restart_policy
-    image: "redis:6.2.14-alpine"
+    image: "docker.io/valkey/valkey:8.1.3-alpine"
```

Comment:
I had `image: eqalpha/keydb:x86_64_v6.3.4` running for months

Thanks for this PR !